### PR TITLE
Adds an example using the js library on the keyValue contract

### PR DESCRIPTION
**Pull request by nkcr**
This PR adds explanations and an example of using our js/cothority library in the browser. It allows one to play with the keyValue contract.

Bellow you have a screenshot of how it looks like.

I also removed the example stored in `cothority_template/html` as this one is now unnecessary.
There are a lot of changed files because I sync'ed the `Conode/` folder from the cothority repo. The demos files are all in the `external/js/` folder.

Screenshot 2019-10-23 at 10 27 08

### CODE REVIEW

**Comment by nkcr** [comment]
@Charrlyine I mention you so you can follow the status of this PR and get notified when the js/example is merged into master.

**Comment by Gilthoniel** [comment]
If you really want to use a different font, you can use Google Fonts that provide embedded solutions like:
```
<link href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i&display=swap" rel="stylesheet"> 
```

**Comment by nkcr** [comment]
> you can use Google Fonts that provide embedded solutions

Sure, I embedded the font in order to have a kind of self-contained app that can be run locally (since it's the most probable use case of it) without relying on the external world. But maybe that doesn't make sense. 
Are you concerned about the extra space the fonts take?

**Comment by Gilthoniel** [comment]
Yeah kind of. It adds something to the template that is not really useful. I guess a template should be as small as possible.

**Comment by nkcr** [comment]
All right, static fonts import removed.

File: external/js/assets/fonts/Roboto/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (Gilthoniel) [inline external/js/assets/fonts/Roboto/LICENSE.txt]: You forgot those ones ?

**Review by Gilthoniel — CHANGES_REQUESTED** [review]

File: external/js/assets/fonts/Roboto/LICENSE.txt (hunk 1 above)
Comment (nkcr) [reply external/js/assets/fonts/Roboto/LICENSE.txt]: 😱

**Review by Gilthoniel — APPROVED** [review]

**Pull request merged**